### PR TITLE
CDAP-4680 log a message when the cdap master has started.

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

File: cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java [hunk 1]
```diff
@@ -179,6 +180,7 @@ public void terminated(Service.State from) {
       @Override
       public void failed(Service.State from, Throwable failure) {
         if (!stopped) {
+          LOG.error("CDAP Master failed to start");
```
Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java]: Do you know if the failure reason is already logged prior to this point?

Comment (albertshau) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java]: yeah, a couple lines above you'll see something like:

```
java.lang.RuntimeException: Unable to start service AppFabricServer [FAILED]: java.util.concurrent.ExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: java.util.concurrent.TimeoutException: Timed out after 30 seconds while waiting to discover the transaction service. Check the logs for the service to see what went wrong.
```

**Comment by sreevatsanraman** [comment]
LGTM

File: cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java (hunk 1 above)
Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java]: :+1:

**Comment by albertshau** [comment]
merging without build since its only log messages

**Pull request merged**